public/sass/pages/_history.scss

```diff
@@ -0,0 +1,268 @@
+// History Table
+.history-table {
+  // .gf-form overrides
+  .gf-form-label { display: none; }
+
+  .gf-form-switch {
+    margin-bottom: 0;
+
+    input + label {
+      height: 3.6rem;
+      width: 110%;
+    }
+
+    input + label::before, input + label::after {
```

Comment:
Thanks again for the refactoring and the feedback, and I'll make sure I don't write CSS like that ever again :)